packages/wonder-blocks-timing/src/index.js

```diff
@@ -21,4 +21,5 @@ export type {
 
 export {SchedulePolicy, ClearPolicy} from "./util/policies.js";
 export {default as withActionScheduler} from "./components/with-action-scheduler.js";
-export {useTimeout} from "./hooks/use-timeout.js";
+export {useScheduledInterval} from "./hooks/use-scheduled-interval.js";
```

Comment:
It seems like we should be exporting `useInterval` too?

Reply:
I'll export it in the next PR.  I want to add docs for it as well.